### PR TITLE
[TRA-4259] Service Map UI improvements

### PR DESCRIPTION
**Pull request by gustavomassa**
[https://www.figma.com/file/1zzIPLSlTTVaaV8oTxGl0d/MIZU?node-id=2689%3A34122)](https://www.figma.com/file/1zzIPLSlTTVaaV8oTxGl0d/MIZU?node-id=2689%3A34122)

TODO: Update the buttons with the new colors and icons
TODO: Align the protocol legend with the buttons

TODO for a new PR: Create a protocol-legend react component that builds the protocol colors dynamically based on a new "Protocol" endpoint that returns an array of registered protocols

### CODE REVIEW

File: ui/src/components/ServiceMapModal/ServiceMapModal.sass [hunk 1]
```diff
@@ -0,0 +1,32 @@
+@import "../../variables.module"
+
+
+.legend-scale ul 
+  margin-top: -29px;
```
Comment (lirazyehezkel) [inline ui/src/components/ServiceMapModal/ServiceMapModal.sass:5]: remove ';'

File: ui/src/components/ServiceMapModal/ServiceMapOptions.ts [hunk 2]
```diff
@@ -1,83 +1,174 @@
...
+const selectedNodeBorderColor = "#205CF5"
+const selectedNodeLabelColor = "#205CF5"
+const selectedEdgeLabelColor = "#205CF5"
+
+function customScaling(min, max, total, value) {
+    if (max === min) {
+        return 0.5;
+    }
+    else {
+        var scale = 1 / (max - min);
```
Comment (lirazyehezkel) [inline ui/src/components/ServiceMapModal/ServiceMapOptions.ts:20]: change to const

File: ui/src/components/ServiceMapModal/ServiceMapModal.tsx [hunk 3]
```diff
@@ -180,33 +190,44 @@ export const ServiceMapModal: React.FC<ServiceMapModalProps> = ({ isOpen, onOpen
...
                         </Button>
+                        </div>
+                        <img src={close} alt="close" onClick={() => onClose()} style={{cursor:"pointer"}}></img>
+                    </div>
                         <Graph
                             graph={graphData}
                             options={ServiceMapOptions}
                         />
+                        <div className='legend-scale'>
+                            <ul className='legend-labels'>
```
Comment (lirazyehezkel) [inline ui/src/components/ServiceMapModal/ServiceMapModal.tsx:222]: Are the colors the same colors as we have on the side of the traffic viewer?
if they are, can we change the color to be const?

File: ui/src/components/ServiceMapModal/ServiceMapOptions.ts [hunk 4]
```diff
@@ -1,83 +1,174 @@
...
+const maxEdgeScaling = maxNodeScaling / 2
+
+const minLabelScaling = 11
+const maxLabelScaling = 16
+const selectedNodeColor = "#0C0B1A"
+const selectedNodeBorderColor = "#205CF5"
+const selectedNodeLabelColor = "#205CF5"
+const selectedEdgeLabelColor = "#205CF5"
+
+function customScaling(min, max, total, value) {
```
Comment (lirazyehezkel) [inline ui/src/components/ServiceMapModal/ServiceMapOptions.ts:15]: can we change all functions to es6 syntax?
for example:
const customScaling = (min, max, total, value) => {

File: ui/src/components/ServiceMapModal/ServiceMapModal.tsx (hunk 3 above)
Comment (AmitUp9) [reply ui/src/components/ServiceMapModal/ServiceMapModal.tsx:222]: I can do that, its just Gustavo wasnt sure if to change the colors to Javier design. its decided not to change at the moment.

**Review by lirazyehezkel — APPROVED** [review]

**Pull request merged**